cities.json

```diff
@@ -4011,6 +4019,25 @@
                         "right": "-76.709"
                     }
                 },
+<<<<<<< HEAD
+                "harrisonburg_virginia": {
+                    "bbox": {
+                        "top": "38.498",
+                        "left": "-78.947",
+                        "bottom": "38.376",
+                        "right": "-78.812"
+                    }
+                },
+=======
+		"hartford_connecticut": {
+                    "bbox": {
+                        "top": "41.818",
+                        "left": "-72.711",
+                        "bottom": "41.710",
+                        "right": "-72.565"
+		    }
+		},
+>>>>>>> eecfdc5fb8bdddadcd66386071cd9983458ed6b2
                 "hellertown_pennsylvania": {
```

Comment:
@KathleenLD there's a little bit of merge conflict junk breaking the tests. 

I'm gonna make a pr to fix right now